Marketplace/emoji-tablet/JSON_files/pizza.json

```diff
@@ -28,6 +28,6 @@
     "lifetime":30,
     "grabbable":true,
     "collisionless":false,
-    "script":"https://hifi-content.s3.amazonaws.com/elisalj/emoji_scripts/behaviors/Edible.js",
+    "script":"file:///D:/hifi-content/Marketplace/emoji-tablet/emoji_scripts/behaviors/Edible.js",
```

Comment:
Can you use `Script.resolvePath()` here too?

Reply:
No, this is a JSON file and the comment explicitly calls out that these aren't final